public/static/js/build.js

```diff
@@ -247,13 +248,10 @@ mciModule.controller('BuildViewController', function($scope, $http, $timeout, $r
         return new Date(x.Task.start_time) > new Date(0) && new Date(x.Task.finish_time) > new Date(0)
       }
     )
-    $scope.totalTimeMS = _.reduce(
-      _.map(finishedOnly,
-        function(x){return new Date(x.Task.finish_time) - new Date(x.Task.start_time)}),
-        function(sum, el){return sum+el},
-        0)
-    };
 
+    $scope.totalTimeMS = _.reduce(_.pluck(finishedOnly, "Task"), function(x, y){return x.time_taken+y.time_taken}, 0) / nsPerMs;
+    };
```

Comment:
This should be indented to the same level as the scope that it closes.

Reply:
done.